src/cascade/dismod/db/wrapper.py

```diff
@@ -0,0 +1,153 @@
+"""
+Creates a file for Dismod-AT to read.
```

Comment:
I liked reading: "stores data in memory until it is explicitly flushed" , would it be useful to put that in the docstring, or is that too "how"?